### DISCUSSION TITLE
2023/collapse/mars/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#53] [General]
# 2023/collapse/mars/

http://shanenull.com/2023/collapse/mars/

**Comment by shane0** [comment]
William Rees recently commented we cannot build a sustainable bio dome on earth so how will it work on mars

Elon should build a bio dome that simulates mars

people should experience it along with train to troubleshoot system failures 

from what I’ve read about mars I would not want to be in the simulator let alone actual mars

**Comment by shane0** [comment]
Neal on terraforming


https://youtu.be/7XdkKMhAdnA?si=sB9REvAhCvnZQbIg
